src/loader.js

```diff
@@ -88,22 +88,26 @@
         return entry.module;
       });
 
+    // @ifdef STRICT
```

Comment:
@guybedford I looked for preprocessors that were not comment based and really only found uglifyjs or babel-dead-code-elimination (which seems to maybe be just a babel 5 thing that is not anymore supported and has a bad github link).

Do you have any ideas for a variable/code based preprocessor that removes the dead code during the preprocessor step? If not, would you prefer the uglifyjs route (where es6-module-loader.src still has all the strict correctness code in it and is the same as es6-module-loader-declarative.src), or would you prefer the comment-based approach that I've already implemented?